config/application.rb

```diff
@@ -23,6 +23,7 @@ class Application < Rails::Application
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
 
+    config.autoload_paths << Rails.root.join('lib')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.